qiita_core/configuration_manager.py

```diff
@@ -168,14 +170,18 @@ def _get_main(self, config):
                              self.working_dir)
         self.max_upload_size = config.getint('main', 'MAX_UPLOAD_SIZE')
         self.require_approval = config.getboolean('main', 'REQUIRE_APPROVAL')
+
         self.plugin_launcher = config.get('main', 'PLUGIN_LAUNCHER')
+        self.plugin_dir = config.get('main', 'PLUGIN_DIR')
+        if not self.plugin_dir:
+            self.plugin_dir = join(expanduser('~'), '.qiita_plugins')
 
         self.valid_upload_extension = [ve.strip() for ve in config.get(
             'main', 'VALID_UPLOAD_EXTENSION').split(',')]
         if (not self.valid_upload_extension or
            self.valid_upload_extension == ['']):
             self.valid_upload_extension = []
-            print 'No files will be allowed to be uploaded.'
+            warnings.warn('No files will be allowed to be uploaded.')
```

Comment:
Before it was just a print - I am ok to make it an error since you will not able to upload any kind of data.